db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ ] In the `seed.rb`, Please add some example data so that any user can easily run your project in local without error. you can add data like this.
```ruby
# db/seed.rb
Greeting.create(:message => "HI!  Buddy!!!")
Greeting.create(:message => "HI!  Cuddy!!!")
Greeting.create(:message => "HI!  Duddy!!!")
Greeting.create(:message => "HI!  Fuddy!!!")
Greeting.create(:message => "HI!  Guddy!!!")
```

Also in the README file please mention the step `rails db:seed` after that so that these data would go to the local database of the user.